### ISSUE TITLE
Links in message bar don't meet color contrast requirements (website-specific)

### ISSUE THREAD

**Issue by cnamal** [OfficeDev/office-ui-fabric-react#10647]
Open documentation of [MessageBar](https://developer.microsoft.com/en-us/fabric#/controls/web/messagebar)
Run [Accessibility Insights](https://accessibilityinsights.io/docs/en/web/overview)
Links have color contrast issues:
![image](https://user-images.githubusercontent.com/9625284/65756576-a3317100-e115-11e9-927c-f3a20d6de629.png)

Additional information:
Title: WCAG 1.4.3: Elements must have sufficient color contrast (.root-114[href$="www\.bing\.com"][target="_blank"])
Tags: Accessibility, WCAG 1.4.3, color-contrast

Issue: Elements must have sufficient color contrast (color-contrast - https://dequeuniversity.com/rules/axe/3.3/color-contrast?application=msftAI)

Target application: Home - Office UI Fabric - https://developer.microsoft.com/en-us/fabric#/controls/web/messagebar

Element path: #MessageBar336 > .innerText-188.ms-MessageBar-innerText[aria-live="polite"] > span > .root-114[href$="www\.bing\.com"][target="_blank"]

Snippet: <a target="_blank" href="www.bing.com" class="ms-Link root-114">Visit our website.</a>

How to fix: 
Fix any of the following:
  Element has insufficient color contrast of 3.96 (foreground color: #0078d4, background color: #dff6dd, font size: 9.0pt (12px), font weight: normal). Expected contrast ratio of 4.5:1

Environment: Chrome version 77.0.3865.90

====

This accessibility issue was found using Accessibility Insights for Web 2.10.0 (axe-core 3.3.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

-----
EDIT by @ecraig12345 -- This issue **only occurs on the website** and needs to be fixed separately (it's not a duplicate). See comment for fix.

**Comment by joschect** [comment]
@cnamal thanks for submitting this. This does look like it's a bug in our demo page. @betrue-final-final do we have some guidance as to what colors these should be? It looks like, at some point, that our colors drifted per this closed issue from a while back #5265

**Comment by ecraig12345** [comment]
I'm going to take this one along with a couple other issues related to MessageBar/Link colors. 

The issue here is that in the website page styles, we have this part to do a more specific override of some bad global styles:
https://github.com/OfficeDev/office-ui-fabric-react/blob/bad2f7e4a975a61f69b47e1792e932b43b17169d/packages/example-app-base/src/components/Page/Page.module.scss#L12-L27

`$linkColor` is `themePrimary`, but in the normal MessageBar styles it uses `themeDark` for links:
https://github.com/OfficeDev/office-ui-fabric-react/blob/bad2f7e4a975a61f69b47e1792e932b43b17169d/packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.styles.ts#L135-L138

So the color in Page styles also needs to be updated to `themeDark`.

(Side note: we may also need a high contrast override for links in Page styles.)

**Comment by ecraig12345** [comment]
Clarification: the themeDark style should be specifically applied to `.ms-MessageBar .ms-Link`
